parser/parser.y

```diff
@@ -5456,6 +5463,9 @@ DateAndTimeType:
 		x := types.NewFieldType(mysql.TypeTimestamp)
 		x.Flen = mysql.MaxDatetimeWidthNoFsp
 		x.Decimal = $2.(int)
+		if x.Decimal == types.UnspecifiedLength{
```

Comment:
Why change this?

Reply:
it's changed by mistake. I'll revert it.

Reply:
@coocood  it's change by mistake. I'll revert it.